helpers/ContentLoader.php

```diff
@@ -43,7 +43,13 @@ public function __construct() {
     public function update() {
         $sourcesDao = new \daos\Sources();
         foreach($sourcesDao->getByLastUpdate() as $source) {
-            $this->fetch($source);
+        	  $last = $source['lastupdate'];
+        	  $wait = $source['waitperiod'];
+        	  if (($last + $wait) < time()){
+	            $this->fetch($source);
+//            } else {
```

Comment:
If you activate these lines you get a feedback on the cliupdate when a feed is skipped. I wasn't sure if that's ok.